cli/commands/actions/diff.ts

```diff
@@ -0,0 +1,75 @@
+import { generateCode, getAllActions, printCodeDiff } from '../../lib/utils'
+import { Change, diffLines } from 'diff'
+import { ACTION_MANIFEST } from '../../manifests'
+import { GetActions200ResponseActionsInner as Action } from 'auth0'
+import { cyan, green, grey, magenta, red } from 'chalk'
+
+type DiffPair = [ActionDefinition, Action | undefined]
+
+export default async function run() {
```

Comment:
This file is very copy-pasted from the Rules version. Once the migration is complete and the actions have been running for a while we can delete the rules version